src/main/java/org/myrobotlab/io/StreamGobbler.java

```diff
@@ -0,0 +1,29 @@
+package org.myrobotlab.io;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class StreamGobbler extends Thread {
+  InputStream is;
+  String type;
+
+  public StreamGobbler(String type, InputStream is) {
+    this.is = is;
+    this.type = type;
+  }
+
+  public void run() {
+    try {
+      InputStreamReader isr = new InputStreamReader(is);
+      BufferedReader br = new BufferedReader(isr);
+      String line = null;
+      while ((line = br.readLine()) != null)
+        System.out.println(type + ">" + line);
+    } catch (IOException ioe) {
+      ioe.printStackTrace();
```

Comment:
should have a proper logger, not stdout.

Reply:
Usually, i'd agree - but this was potentially part of Client.java which is supposed to be the very minimal java app capable of interfacing with mrl.  Picocli and wasync are the "only" dependencies - and its a console app - so I've been on the fence to including logging vs stdin/stdout.   At the current time locally Client doesnt use this file - and I'm not sure if it will stay.